src/Bugsnag/Middleware.cs

```diff
@@ -1,5 +1,7 @@
 using System.Linq;
+using System.Reflection;
```

Comment:
Guessing it's a bit different on .NET, but reflection is very slow on Java. I'm assuming that won't be an issue here as all it seems to be doing is type checks?